src/katsdpk8spoc/sdpcontroller.py

```diff
@@ -347,6 +366,10 @@ def get_config():
 def main():
     config = get_config()
     app = web.Application()
+    aiohttp_jinja2.setup(
+        app,
+        loader=jinja2.FileSystemLoader("/src/katsdpk8spoc/")
```

Comment:
I think moving templates to another directory is better.

Reply:
Sounds like a good idea